Controller/UserController.php

```diff
@@ -122,12 +125,12 @@ public function editAction(Request $request)
         /** @var \Symfony\Bundle\FrameworkBundle\Translation\Translator $translator */
         $translator = $this->get('translator');
 
-        $username = (string) $request->get('username');
+        $id = (string) $request->get('id');
```

Comment:
We normally put the id in the method signature

`public function editAction(Request $request, $id)` 

This way, you can remove this line. The cast to string is btw not correct too ;)